package.json

```diff
@@ -14,7 +14,8 @@
     "start:main": "cross-env NODE_ENV=development electron -r ts-node/register/transpile-only ./src/main/main.ts",
     "start:renderer": "cross-env NODE_ENV=development TS_NODE_TRANSPILE_ONLY=true webpack serve --config ./.erb/configs/webpack.config.renderer.dev.ts",
     "test": "jest",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "start-express": "nodemon ./src/main/expressServer.ts"
```

Comment:
Here is what I have to say about this.

- `nodemon` is tool which helps the developer by automatically restarting the node application when file changes in the directory are detected.
    - Once the app is built the files will not be changing.
    - Starting the server via `scripts` is something we can do now in development, however when we build the app these scripts will not be there and thus the server will not be running.
    - The solution to this is to start the server using a childprocess. Look at the how the python server is ran here: [main/pyServer.ts - line 4](https://github.com/patrickbrett/mlvet/blob/develop/src/main/pyServer.ts#L4)
    - One thing I recommend is using the [cwd option in the child_process.spawn function](https://nodejs.org/api/child_process.html#child_processspawncommand-args-options) (I have not yet updated the code to use it. Changing directories is not the correct way to go about it)

Reply:
@Taz17 do we need a child process though? Unless I'm missing something we should just be able to start the server in the same process as the electron back end

Reply:
I remember having a chat about that and it was mentioned that running the server through the electron/nodejs backend was buggy.

Just to be cautious, its better to have @zwan0202 @itsMoxMox to verify that they have tried running the server from the electron/nodejs backend and what the outcome was.

Reply:
Sorry for the late reply.

I refactored the express server and let it run as a child process. The server runs fine and I can get response from it. 

But every time the project start, it will pop up an error window says 'Connection refused'. 

I put server and route file as standard `js` files, otherwise Node will found an `import` statement in a file that it does not consider an ECMAScript (ES) module. I don't want to set the module system at the project level. I tried to use the `.mts` and `.cts` extensions, but it not working and not showing any error or warning message.